sale_procurement_analytic/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Sale Procurement Analytic
+=========================
+
+This module sets analytic account procurement from sale order.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Settings > Configuration > Accounting > Analytic Accounting*.
+#. Enable *Analytic Accounting for Sales*.
+
+Usage
+=====
+
+#. Go to *Sales -> Sales Orders* and create new SO witch contains product/s
+   with *Suppliers* info and *Make To Order* *Route* are set.
+#. Set *Contract / Analytic* field.
+#. *Confirm Sale*
+
```

Comment:
Go to Warehouse -> Schedulers -> Procurements and check *Analytic Account* in *Extra Information* tab